RELEASE.md

```diff
@@ -0,0 +1,25 @@
+# Release
+
+## Releasing a major version.
+
+Rename UNRELEASED to a concrete <version> and create a PR with commit: Release <version>. 
+Branch name should match the <version> set in CHANGELOG. 
+Additionally tag that same commit with 0 as a patch version (if branch is v1.1 - tag is v1.1.0).
+
+## Releasing a minor version.
+
+### Latest
+
+Rename UNRELEASED to a concrete <version>. If previous major was v1.1.0, new version will
+be v1.1.1. 
+Commit changes, create pr, and create a tag.
+Rebase released branch onto develop.
+
+This is a simplified workflow to avoid cherry-picks and maintaining changelog in several branches.
```

Comment:
Just to see if I understood correctly:

Lets say we find a bug in version `v1.1.2`. Shall I create a PR to fix that bug targeting the `v1.1` branch and update `CHANGELOG.md` with the intention that this one PR will be `v1.1.3`? I assume after the merge of that PR tag the `v1.1` branch with `v1.1.3` and create a PR to also merge it into develop?

Reply:
if v1.1 is latest then and it didn't diverge from develop, then we merge changes to develop and rebase v1.1 on top of develop.

if v1.1 diverged and we can't rebase (or due to any other reasons) then change merged separately to v1.1, and changelog has to be adjusted separately